Vagrantfile

```diff
@@ -5,8 +5,8 @@
 # vi: set ft=ruby :
 
 # UPDATE THESE FOR YOUR PROJECT
-project_name = "pyzappi"
-hosts_aliases = %w(pyzappi.localdomain "pyzappi.web-masons.org")
+project_name = "warcry"
+hosts_aliases = %w(warcry.localdomain "warcry.web-masons.org")
```

Comment:
This is the Pyzapi project, so you don’t want to change the name of the project to something else. If you’re making a copy of the project, you could do that, but we don’t want to change the base project.